db/migrate/20230607202910_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title, limit: 225, null: false # 255 max chars
+      t.text :text, null: false
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] From the ERD, I could see that the user model was referenced as author_id as a relationship with other models. So to ensure that your schema matches the ERD, it will be better to use author_id to reference users.
e.g

In your post migration file you should have  something like :
`t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index `